src/simulation/simulation_sequence.jl

```diff
@@ -192,12 +192,49 @@ function _attach_feedforwards(models::SimulationModels, feedforwards)
     return ff_dict
 end
 
-@doc raw"""
+"""
     SimulationSequence(
-                        models::SimulationModels,
-                        feedforward::Dict{Symbol, <:AbstractAffectFeedforward}
-                        ini_cond_chronology::Dict{Symbol, <:FeedforwardChronology}
-                        )
+        models::SimulationModels,
+        feedforward::Dict{String, Vector{<:AbstractAffectFeedforward}}
+        ini_cond_chronology::InitialConditionChronology
+    )
+
+Construct the simulation sequence between decision and emulation models.
+
+# Arguments
+
+  - `models::SimulationModels`: Vector of decisions and emulation models.
```

Comment:
Why are you always adding whitespace before the dashes? I'm sure it doesn't matter, but I don't think that is standard practice.

Reply:
This is done by the formatter